src/jqLite.js

```diff
@@ -288,6 +288,8 @@ function JQLite(element) {
 
   if (argIsString) {
     jqLiteAddNodes(this, jqLiteParseHTML(element));
+  } else if (isFunction(element)) {
```

Comment:
It's probably not a problem, but it can't do much damage either (how many times would one call `jqLite(fn)`?).
So if jQuery has it this way, let's leave it as is.